examples_test.go

```diff
@@ -16,6 +16,8 @@ type SampleUser struct {
 	BirthDate   time.Time              `json:"birth_date,omitempty" jsonschema:"oneof_required=date"`
 	YearOfBirth string                 `json:"year_of_birth,omitempty" jsonschema:"oneof_required=year"`
 	Metadata    interface{}            `json:"metadata,omitempty" jsonschema:"oneof_type=string;array"`
+	IPAddress   interface{}            `json:"ipAddress,omitempty" jsonschema:"oneof_ref=#/$defs/ipv4;#/$defs/ipv6"`
+	IPAddresses []interface{}          `json:"ipAddresses,omitempty" jsonschema:"oneof_ref=#/$defs/ipv4;#/$defs/ipv6"`
```

Comment:
I appreciate these are only samples, but I'd still stick to the common pattern of `ip_address`.

Reply:
Sorry for the delay.  Done.  PTAL.